Gopkg.toml

```diff
@@ -8,7 +8,7 @@
 
 [[constraint]]
   name = "github.com/grpc-ecosystem/grpc-gateway"
-  version = "1.3.1"
+  version = "1.4.1"
```

Comment:
How much of a difference does this make?

Reply:
it varies from repo to repo, however I would make it consistent with gentool